README.md

```diff
@@ -1,2 +1,190 @@
 # EventManager
+[![Build Status](https://travis-ci.org/nzediegwu1/EventManager.svg?branch=develop)](https://travis-ci.org/nzediegwu1/EventManager) [![Coverage Status](https://coveralls.io/repos/github/nzediegwu1/EventManager/badge.svg?branch=develop)](https://coveralls.io/github/nzediegwu1/EventManager?branch=develop)
+ [![Maintainability](https://api.codeclimate.com/v1/badges/8413c3ad5cdf27f79e38/maintainability)](https://codeclimate.com/github/nzediegwu1/EventManager/maintainability)
+
 Given you manage an events center, this app will help you accept applications to use your center/facilities, and will either decline events when the proposed day is already taken, or suggest an available day
+
+###### App's Documentation: https://eventmanager29.herokuapp.com/docs
+<hr>
+
+## TECHNOLOGIES USED
+
+  * Front-end: React/Redux + Bootstrap (Yet to be Implemented)
+  * Back-end: Node/Expressjs + Sequelize/Postgres
+  * Libraries: nodemon, Babel, eslint, etc.
+  * Test: Mocha/Chai
+
+
+## Setup guide: Clone this repository. To install all dependencies, run;
+
+
+
+``` npm install  ```
+
+
+
+To start server, run:
+
+
+
+``` npm run start:dev  ```
```

Comment:
This command doesn't work (anymore)? Update to match what you have in your package.json

Reply:
Will modify. Thanks for the feedback